MetamodelLD.md

```diff
@@ -218,7 +218,7 @@ Deze paragraaf is een aanvulling op de paragraaf [[[#specificatie-metagegevens]]
 | alias | [`mim:alias`](http://bp4mc2.org/def/mim#alias) | owl:DatatypeProperty | [grondslag](#metagegeven-alias) |
 | attribuut | [`mim:attribuut`](http://bp4mc2.org/def/mim#attribuut) | owl:ObjectProperty | [grondslag](#metagegeven-heeft-attribuut) |
 | authentiek | [`mim:authentiek`](http://bp4mc2.org/def/mim#authentiek) | owl:ObjectProperty | [grondslag](#metagegeven-authentiek) |
-| basis-URI | [`mim:basisUri`](http://bp4mc2.org/def/mim#basisUri) | owl:DatatypeProperty | [grondslag](#metagegeven-basis-uri) |
+| basis-URI | [`mim:basisUri`](http://bp4mc2.org/def/mim#basisUri) | owl:ObjectProperty | [grondslag](#metagegeven-basis-uri) |
```

Comment:
Het is mij nog even de vraag hoe we hier mee om willen gaan. Mijn insteek was dat de basis uri als literal gezien werd. 
Bijvoorbeeld:
`<#mim> mim:basisUri "http://modellen.mim-standaard.nl/def/mim#"^^xsd:anyURI` of
`<#mim> mim:basisUri "http://modellen.mim-standaard.nl/def/mim#"^^xsd:string`

Het is ook niet zo dat deze waarde de facto de ontologie (een object) identificeert. Is de baseURI dus een individual? Dat is de requirement voor het gebruik van owl:Objectproperty. Wat geldt voor waarden als `urn:modelelement:imkad:recht"

Reply:
In het LD metamodel zou ik daar wel vanuit gaan

Reply:
Ik zou inderdaad ook uitgaan van een literal. Zie bijvoorbeeld ook https://www.w3.org/TR/shacl/#sparql-prefixes waar hetzelfde wordt gedaan (dus idd een xsd:anyURI)